<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPump">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>水泵</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTurbine">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>汽轮机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimulator">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开启仿真模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRunMode">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开启运行模式</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
